renovate.json

```diff
@@ -27,8 +28,7 @@
       "excludePackagePatterns": [
         "^github.com/openstack-k8s-operators"
```

Comment:
not sure I understand this excludePackagePatterns and the above matchPackageNames. Doesn't the above include it, but here we exclude it?

Reply:
I've reworked this now to be explicit with the packages we are disabling.